xarray/core/dataset.py

```diff
@@ -2788,6 +2959,26 @@ def tail(
             The keyword arguments form of ``indexers``.
             One of indexers or indexers_kwargs must be provided.
 
+        Examples
+        --------
```

Comment:
`.head` accepts number of elements as a single integer for all dimensions or as a dict which is specific to certain dimensions. It would be nice to have examples that cover both options.